simulation/runSim.sh

```diff
@@ -1,8 +1,8 @@
-COPPELIA_SIM_SCRIPT=~/dev/coppelia/coppeliaSim.sh
+COPPELIA_SIM_SCRIPT=~/docker-build/coppelia/coppeliaSim.sh
 SERVER="comm_server_scene.ttt"
 echo "Starting CoppeliaSim"
 
-SERVER_LOCATION=$(find /home/duke/dev -name $SERVER | head -1)
+SERVER_LOCATION=$(find /root -name $SERVER | head -1)
```

Comment:
Do you mean `/root/dev` for this, since we still have a dev folder inside root?

Reply:
/root/dev will also work, and would probably be better - the scene file will always be in the git repo.

Reply:
then, could you also change that as well?